### PR TITLE
don't allow users to rate if cannot rate (bug 1126529/1127388)

### PR DESCRIPTION
**Pull request by ngokevin**

### CODE REVIEW

File: src/media/js/reviews.js [hunk 1]
```diff
@@ -83,7 +83,14 @@ define('reviews',
...
         function onLoginSuccess() {
             var reviewButton = document.querySelector('.review-button');
-            if (caps.widescreen()) {
+
+            if (!reviewButton) {
+                setTimeout(function() {
+                    notification.notification({
+                        message: gettext('Sorry, you must purchase this app before reviewing'),
+                    });
+                });
```
Comment (spasovski) [inline src/media/js/reviews.js]: No time param?

Comment (ngokevin) [inline src/media/js/reviews.js]: Nope, just need to bump it ahead of the login notification. I will comment

File: src/templates/_includes/reviews_sidebar.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
           {{ _('Edit your review') }}
         </a>
       </li>
-    {% else %}
+    {% elif not (user.logged_in() and not response.user.can_rate) %}
```
Comment (spasovski) [inline src/templates/_includes/reviews_sidebar.html]: Truth table checks out...tricky to read though.

**Comment by spasovski** [comment]
Looks good r+

File: src/templates/_includes/reviews_sidebar.html (hunk 2 above)
Comment (ngokevin) [inline src/templates/_includes/reviews_sidebar.html]: Yeah...saves template though

**Comment by ngokevin** [comment]
thanks

**Pull request merged**